src/mcp_agent/workflows/llm/multipart_converter_azure.py

```
@@ -0,0 +1,360 @@
+            return TextContentItem(text=text)
+
+        text = get_text(resource)
+        if text:
+            return TextContentItem(text=text)
+
+        if isinstance(resource.resource, BlobResourceContents) and hasattr(
+            resource.resource, "blob"
+        ):
+            blob_length = len(resource.resource.blob)
+            return TextContentItem(
+                text=f"Embedded Resource {getattr(uri, '_url', '')} with unsupported format {mime_type} ({blob_length} characters)"
```

Pylint (3.3.7)

[convention] Line too long (131/100)

(C0301)